tests/integration-larvavel.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+# Inspired by https://github.com/nunomaduro/larastan/blob/669b489e10558bd45fafc2429068fd4a73843802/tests/laravel-test.sh
+#
+# Create a fresh Laravel installation, install our package in it and run some
+# basic tests to ensure everything works.
+#
+# This script is meant to be run on Travis CI
+
+echo "Prevent shallow repository error"
+git fetch --unshallow
+
+echo "Install Laravel"
+travis_retry composer create-project --quiet --prefer-dist "laravel/laravel" laravel
+cd laravel
+
+echo "Add package from source"
+sed -e 's|"type": "project",|&\n"repositories": [ { "type": "vcs", "url": "../" } ],|' -i composer.json
+travis_retry composer require --dev "rebing/graphql-laravel:dev-master#${TRAVIS_COMMIT}"
+
+echo "Publish vendor files"
+php artisan vendor:publish --provider="Rebing\GraphQL\GraphQLServiceProvider"
+
+echo "Make GraphQL ExampleQuery"
+php artisan make:graphql:query ExampleQuery
+
+echo "Add ExampleQuery to config"
+sed -e "s|// 'example_query' => ExampleQuery::class,|\\\App\\\GraphQL\\\Query\\\ExampleQuery::class,|" -i config/graphql.php
+
+echo "Start Webserver"
+php -S 127.0.0.1:8001 -t public >/dev/null 2>&1 &
+sleep 2
```

Comment:
Not sure, could be `sleep 1`. I couldn't find a ready-available shell helper to wait until the web server is available but since startup time should be way <1s I figured let' make it simple.